src/Groups/libraries/atlasgroups.jl

```diff
@@ -2,6 +2,7 @@ export
     all_atlas_group_infos,
     atlas_group,
     atlas_program,
+    atlas_subgroup,
```

Comment:
shouldn't it also be referenced in one of the markdown files in the `doc` dir?

Reply:
Yes. Thanks.